docs/sources/mimir/set-up/jsonnet/_index.md

```diff
@@ -1,6 +1,8 @@
 ---
 aliases:
   - ../deploying-grafana-mimir/jsonnet/
+  - ../../operators-guide/deploy-grafana-mimir/jsonnet/
+  - ../operators-guide/deploy-grafana-mimir/jsonnet/
```

Comment:
Are both of these required?

Reply:
I do not know. Is it unsafe to leave as is?